pkg/proxy/routes.go

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package proxy
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"syscall"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/linux_defaults"
+	"github.com/cilium/cilium/pkg/datapath/linux/route"
+)
+
+const (
+	// Mark/mask set by the bpf datapath to denote packets destined to the proxy.
+	tproxyMark = 0x200
+	tproxyMask = 0xf00
+
+	// Routing table for redirecting packets to the proxy through the local stack.
+	proxyRoutingTable = 2004
+)
+
+var (
+	// Routing rule for traffic to proxy.
+	tproxyRule = route.Rule{
+		Priority: 9,
```

Comment:
I don't know how to feel about this one. I know this is just taken from init.sh. But it is an oddly specific number.

Reply:
Would you like to see this assigned to a constant? Or a rationale behind the value 9?

Reply:
I would add a comment at least, if it makes sense inline there or else on a constant. That comment would ideally have a rationale, but if we can't find one it would be nice to know that as well. If its just arbitrary between 0 and 100 at the low side of things then someone doesn't have to be afraid to touch it in fear of some Indiana Jones trap springing at any moment.